pnpm-lock.yaml

```diff
@@ -7996,6 +8297,7 @@ packages:
       number-to-bn: 1.7.0
       randombytes: 2.1.0
       utf8: 3.0.0
+    dev: true
```

Comment:
Are these intentional?

Reply:
We haven't updated lock file since the dependencies are updated.